distros/kubernetes/nvsentinel/charts/janitor/templates/configmap.yaml

```
@@ -66,3 +66,17 @@ data:
+      enabled: {{ if (hasKey .Values.config.controllers.gpuReset "enabled") }}{{ .Values.config.controllers.gpuReset.enabled }}{{ else }}true{{ end }}
+      timeout: {{ .Values.config.controllers.gpuReset.timeout | default .Values.config.timeout | default "25m" }}
+      manualMode: {{ .Values.config.manualMode | default false }}
+      {{- if .Values.config.controllers.gpuReset.cspProviderHost }}
+      cspProviderHost: {{ .Values.config.controllers.gpuReset.cspProviderHost }}
+      {{- end }}
+      resetJobImage: "{{ .Values.config.controllers.gpuReset.resetJobImage.repository }}:{{ .Values.config.controllers.gpuReset.resetJobImage.tag | default ((.Values.global).image).tag | default .Chart.AppVersion }}"
+      {{- if .Values.config.controllers.gpuReset.serviceManager }}
+      serviceManager: {{ .Values.config.controllers.gpuReset.serviceManager }}
+      {{- end }}
+
+
```

YAMLlint (1.38.0)

[warning] too many blank lines (2 > 0)

(empty-lines)